WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -93,4 +106,49 @@ public class PeopleViewController: UITableViewController, NSFetchedResultsContro
         let person = Person(managedPerson: managedPerson)
         return person
     }
+    
+    
+    // MARK: - UIViewControllerRestoration
+    
+    public class func viewControllerWithRestorationIdentifierPath(identifierComponents: [AnyObject], coder: NSCoder) -> UIViewController? {
+        
+        let context = ContextManager.sharedInstance().mainContext
+        
+        guard let blogID = coder.decodeObjectForKey(RestorationKeys.blog) as? String,
+            let objectURL = NSURL(string: blogID),
+            let objectID = context.persistentStoreCoordinator?.managedObjectIDForURIRepresentation(objectURL),
+            let restoredBlog = try? context.existingObjectWithID(objectID) else
+        {
+            return nil
+        }
+        
+        guard let blog = restoredBlog as? Blog else {
```

Comment:
Is there a reason this is separate from the `guard` statements above? You could append it to the list of `let`s there, or even add the `as?` to the final statement:

`let blog = (try? context.existingObjectWithID(objectID)) as? Blog else {`